client/templates/tweet.html

```diff
@@ -0,0 +1,26 @@
+<md-list-item class="md-3-line">
```

Comment:
This should be the version pulled from the client tweet wall rather than the admin tweet wall since the client tweet wall has the fixes in pull request 43 that the admin wall doesn't.